src/db/seed.py

```
@@ -1,196 +1,303 @@
+                games_played=15,
+                stats=p["stats"],
+                fantasy_points={"ppr": p["stats"].get("fantasy_ppr", 0)}
+            )
+            session.add(stats)
     session.flush()
     print("NFL players seeded")
 
 
-def seed_nba_players(session, nba_league):
-    """Seed sample NBA players with stats."""
+def seed_nba_players(session, league_id: str):
```

Ruff (0.14.14)

Unused function argument: `league_id`

(ARG001)